### PR TITLE
Removing flex_categories from csd1 and csd3 (-Ram)

### PR DESCRIPTION
**Pull request by balderdash**
Also grabbed `cspassessment.script` at Baker's request.

### CODE REVIEW

File: dashboard/config/scripts/cspassessment.script [hunk 1]
```diff
@@ -3,9 +3,3 @@ login_required true
 stage 'CS Principles Culminating Assessment'
 level 'CSP_Final_instructions'
 assessment 'CSP Culminating Assessment'
-
-stage 'CSP Student Post Survey Staging'
-assessment 'CSP post-assessment survey'
-
-stage 'Participate in a longitudinal study!'
-level 'Longitudinal study description'
```
Comment (Bjvanminnen) [inline dashboard/config/scripts/cspassessment.script:11]: are these changes intentional?

**Comment by balderdash** [comment]
Closing in favor of https://github.com/code-dot-org/code-dot-org/pull/14582
